### ISSUE TITLE
ReactDOM is not defined in configuration wizard

### ISSUE THREAD

**Issue by generall** [allenai/allennlp#2480]
**Describe the bug**
Configuration wizard GUI not working. It shows only single title on web page

**To Reproduce**
Steps to reproduce the behavior
1. install allennlp into clear virtualenv.
2. Run `allennlp configure`
3. Open browser `http://localhost:8123/`
4. See empty screen

**Expected behavior**
See configuration wizard with input forms e.t.c.

**System (please complete the following information):**
 - OS: Ubuntu 18.04
 - Python version: 3.6.7
 - AllenNLP version: 0.8.1
 - Browser: Google Chrome Version 70.0.3538.77 (Official Build) (64-bit)

**Additional context**
Console in browser shows following error report:
```
Access to script at 'https://unpkg.com/react-dom@16.7.0/umd/react-dom.production.min.js' (redirected from 'https://unpkg.com/react-dom@16/umd/react-dom.production.min.js') from origin 'http://localhost:8123' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

runScripts @ babel.js:61666
transformScriptTags @ babel.js:336
(anonymous) @ babel.js:327
Inline Babel script:679 Uncaught ReferenceError: ReactDOM is not defined
    at <anonymous>:919:1
    at run (babel.js:61531)
    at check (babel.js:61597)
    at loadScripts (babel.js:61638)
    at runScripts (babel.js:61668)
    at transformScriptTags (babel.js:336)
    at babel.js:327
```

**Comment by joelgrus** [comment]
I believe this is the issue fixed by #2476. I guess we need to do a new release.

**Comment by HarshTrivedi** [comment]
@joelgrus  Unrelated to this, but before doing a release, it would be better to have #2395 merged.
 In it, I have removed unncessary public extension method introduced in #2374. If #2395 is merged after the release, we might have to take trouble of deprecating them first.
